lib/mayaUsd/ufe/UsdUndoDeleteCommand.cpp

```diff
@@ -85,8 +87,10 @@ void UsdUndoDeleteCommand::execute()
     auto        targetPrimSpec = stage->GetEditTarget().GetPrimSpecForScenePath(_prim.GetPath());
 
     if (hasLayersMuted(_prim)) {
-        TF_WARN("Cannot remove prim because there are muted layers.");
-        return;
+        const std::string error = TfStringPrintf(
+            "Cannot remove prim \"%s\" because there are muted layers.", _prim.GetPath().GetText());
+        TF_WARN("%s", error.c_str());
+        throw std::runtime_error(error);
```

Comment:
Maya catches exceptions from UFE command, so this change is fine. It is required so that UFE composite command can be properly failed when a sub-command fails. For example, the group and reparent UFE commands are actually composite commands and used to be able to half-fail without even knowing they had failed.